.github/workflows/workflow.yml

```diff
@@ -0,0 +1,72 @@
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+env:
```

Comment:
The top-level env vars section pertains only to GitLab due to a bug in the CI/CD system. Since GitHub has Docker built-in, you should delete the top-level `env` and `services` objects.